OZON.ru.js

```diff
@@ -406,24 +131,18 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"notes": [],
-				"tags": [],
-				"seeAlso": [],
-				"attachments": [
-					{
-						"title": "OZON.ru Link",
-						"mimeType": "text/html",
-						"snapshot": false
-					}
-				],
-				"title": "Язык программирования C++",
-				"language": "ru",
-				"publisher": "Бином, Невский Диалект",
-				"ISBN": "5-7940-0031-7, 5-7989-0127-0, 0-201-88954-4",
-				"date": "1999-1-1",
-				"numPages": "991",
+				"date": "1999",
+				"ISBN": "9785794000313",
```

Comment:
Well, yes, `ZU.cleanISBN` will return a single ISBN. You would have to split them up first, then clean them individually and re-join. Actually, you don't even have to worry about cleaning them up in the translator, Zotero should automatically clean all of them up during import (I don't recall if they have to be generally clean from non-ISBN-related text).You can store multiple ISBNs in the same field separated by space, commas, semicolons, something else. Zotero will clean all of that up and de-duplicate any ISBN-10/ISBN-13s. I don't think we take full advantage of this yet, but a full list of ISBNs could be used for detecting duplicates. Otherwise, we just pass the first ISBN to the citation processor.